CONTRIBUTING.md

```diff
@@ -1,18 +1,20 @@
 ![Mizu: The API Traffic Viewer for Kubernetes](assets/mizu-logo.svg)
-# CONTRIBUTE
+
+# Contributing to Mizu
```

Comment:
Can we move this file to .github folder?

Reply:
@IgorGov the right place for this file is the root directory as far as I know. See the GitHub's related doc page https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Reply:
@IgorGov similarly moved it to `.github/` directory.